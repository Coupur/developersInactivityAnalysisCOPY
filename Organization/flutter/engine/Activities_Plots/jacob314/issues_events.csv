id,date,event,creator_login
2442400587,2019-06-26 22:38:07,referenced,jacob314
2435055048,2019-06-24 17:42:08,mentioned,jacob314
2435055049,2019-06-24 17:42:08,subscribed,jacob314
2379419295,2019-05-30 21:54:38,mentioned,jacob314
2379419296,2019-05-30 21:54:38,subscribed,jacob314
2376731270,2019-05-29 23:51:59,mentioned,jacob314
2376731271,2019-05-29 23:51:59,subscribed,jacob314
2348341837,2019-05-17 05:33:23,review_requested,jacob314
2356476733,2019-05-21 14:59:20,head_ref_force_pushed,jacob314
2356545802,2019-05-21 15:18:53,head_ref_force_pushed,jacob314
2332458477,2019-05-10 08:41:30,review_requested,jacob314
2353841606,2019-05-20 17:40:29,mentioned,jacob314
2353841607,2019-05-20 17:40:29,subscribed,jacob314
2353862635,2019-05-20 17:49:29,closed,jacob314
2218552599,2019-03-21 00:37:47,review_requested,jacob314
2218552600,2019-03-21 00:37:47,review_requested,jacob314
2260546707,2019-04-08 18:05:47,mentioned,jacob314
2260546709,2019-04-08 18:05:47,subscribed,jacob314
1908292770,2018-10-17 01:00:54,review_requested,jacob314
2073469091,2019-01-15 02:06:41,closed,jacob314
1644673679,2018-05-24 16:07:20,mentioned,jacob314
1644673681,2018-05-24 16:07:20,subscribed,jacob314
1582728408,2018-04-19 01:07:01,closed,jacob314
1474467073,2018-02-14 18:56:22,referenced,jacob314
1474467076,2018-02-14 18:56:22,merged,jacob314
1474467081,2018-02-14 18:56:22,closed,jacob314
1416128289,2018-01-09 17:30:39,review_requested,jacob314
1416128292,2018-01-09 17:30:39,review_requested,jacob314
1418621103,2018-01-10 21:52:18,review_requested,jacob314
1418907646,2018-01-11 01:52:59,head_ref_force_pushed,jacob314
1427571995,2018-01-17 05:19:37,head_ref_force_pushed,jacob314
1429808613,2018-01-18 07:19:09,head_ref_force_pushed,jacob314
1448896876,2018-01-30 16:12:44,head_ref_force_pushed,jacob314
1455579709,2018-02-02 17:56:36,head_ref_force_pushed,jacob314
1455698144,2018-02-02 19:03:18,head_ref_force_pushed,jacob314
1458588340,2018-02-05 18:51:22,head_ref_force_pushed,jacob314
1462818110,2018-02-07 16:58:17,head_ref_force_pushed,jacob314
1470283064,2018-02-12 20:18:18,head_ref_force_pushed,jacob314
1470714988,2018-02-13 01:14:12,head_ref_force_pushed,jacob314
1470715436,2018-02-13 01:14:33,head_ref_force_pushed,jacob314
1470741867,2018-02-13 01:40:18,head_ref_force_pushed,jacob314
1472108690,2018-02-13 17:12:10,head_ref_force_pushed,jacob314
1472114787,2018-02-13 17:15:12,head_ref_force_pushed,jacob314
1472475827,2018-02-13 20:33:24,referenced,jacob314
1472475828,2018-02-13 20:33:24,merged,jacob314
1472475833,2018-02-13 20:33:24,closed,jacob314
1472476020,2018-02-13 20:33:28,head_ref_deleted,jacob314
