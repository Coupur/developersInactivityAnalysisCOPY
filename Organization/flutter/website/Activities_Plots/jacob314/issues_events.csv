id,date,event,creator_login
2455915372,2019-07-02 23:16:42,mentioned,jacob314
2455915373,2019-07-02 23:16:42,subscribed,jacob314
2458295518,2019-07-03 17:20:45,mentioned,jacob314
2458295519,2019-07-03 17:20:45,subscribed,jacob314
2166355273,2019-02-26 22:28:45,mentioned,jacob314
2166355275,2019-02-26 22:28:45,subscribed,jacob314
2134939078,2019-02-12 21:01:13,mentioned,jacob314
2134939080,2019-02-12 21:01:13,subscribed,jacob314
2030460379,2018-12-17 14:53:15,mentioned,jacob314
2030460382,2018-12-17 14:53:15,subscribed,jacob314
2003678099,2018-12-04 09:12:19,mentioned,jacob314
2003678103,2018-12-04 09:12:19,subscribed,jacob314
2006028998,2018-12-05 02:58:47,assigned,jacob314
2234561178,2019-03-27 21:30:16,closed,jacob314
1972448364,2018-11-16 23:22:08,mentioned,jacob314
1972448365,2018-11-16 23:22:08,subscribed,jacob314
1937197401,2018-10-31 08:33:13,assigned,jacob314
2401507726,2019-06-10 16:32:33,unassigned,jacob314
1826415271,2018-09-04 21:45:03,mentioned,jacob314
1826415272,2018-09-04 21:45:03,subscribed,jacob314
1512838343,2018-03-09 08:49:13,mentioned,jacob314
1512838344,2018-03-09 08:49:13,subscribed,jacob314
1497343672,2018-02-28 18:14:03,mentioned,jacob314
1497343673,2018-02-28 18:14:03,subscribed,jacob314
1514175844,2018-03-09 21:32:47,mentioned,jacob314
1514175845,2018-03-09 21:32:47,subscribed,jacob314
1514241216,2018-03-09 22:15:50,closed,jacob314
1488026982,2018-02-22 22:29:11,review_requested,jacob314
1488026983,2018-02-22 22:29:11,review_requested,jacob314
1488026984,2018-02-22 22:29:11,review_requested,jacob314
1489381919,2018-02-23 16:02:27,referenced,jacob314
1489381920,2018-02-23 16:02:27,merged,jacob314
1489381921,2018-02-23 16:02:27,closed,jacob314
1485095934,2018-02-21 16:40:36,mentioned,jacob314
1485095935,2018-02-21 16:40:36,subscribed,jacob314
1190479546,2017-08-02 23:00:09,review_requested,jacob314
1190502805,2017-08-02 23:25:45,review_requested,jacob314
1190523655,2017-08-02 23:49:37,referenced,jacob314
1190523656,2017-08-02 23:49:37,merged,jacob314
1190523657,2017-08-02 23:49:37,closed,jacob314
