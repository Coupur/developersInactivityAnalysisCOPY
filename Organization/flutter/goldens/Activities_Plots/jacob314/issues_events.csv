id,date,event,creator_login
1828528578,2018-09-05 18:00:37,review_requested,jacob314
1828584746,2018-09-05 18:26:47,referenced,jacob314
1828584749,2018-09-05 18:26:47,merged,jacob314
1828584758,2018-09-05 18:26:47,closed,jacob314
1828584899,2018-09-05 18:26:51,head_ref_deleted,jacob314
1814748094,2018-08-29 05:50:55,head_ref_force_pushed,jacob314
1816701337,2018-08-29 20:59:05,referenced,jacob314
1816701339,2018-08-29 20:59:05,merged,jacob314
1816701342,2018-08-29 20:59:05,closed,jacob314
1813655872,2018-08-28 17:11:54,closed,jacob314
